### PR TITLE
Documenting Database Structure

### LINKED ISSUE #11: Documentation database structure
Add on the readme the database structure and the explanation

possible tables:

### tags (list of tags)
properties and type: id (serial), name (varchar), active (Boolean)
 
### joint table which link the tags to the categories

I think we should start with just the tags table and add the other on the next iteration. Any thoughts or other tables to add @JMurphyWeb @mk4111 ?

### PR DESCRIPTION
**Pull request by mk4111**
Adds description of intended/preliminary thoughts on database structure, and our reasons behind the fields chosen per table.

### CODE REVIEW

**Comment by mk4111** [comment]
@SimonLab - I think this is ready! Could you please have a look and let me know what you think? Thank you! :D

File: README.md [hunk 1]
```diff
@@ -72,6 +72,48 @@ function (request, reply) {
...
+
+Each tag will detail:
+
+| field  | data-type    |
+| --------------------- |
+| id     | serial       |
+| name   | varchar(50)  |
+| active | boolean      |
+
+### **Tag-Categories**
```
Comment (mk4111) [inline README.md]: _SO_ true...

File: README.md [hunk 2]
```diff
@@ -72,6 +72,48 @@ function (request, reply) {
...
+| active | boolean      |
+
+### **Categories-Tags**
+This table will detail the relationships between a tag and a group; i.e. it will list which _tag_ belongs in which _group_.
+
+Each group will detail:
+
+| field            | data-type    |
+| ------------------------------- |
+| category- id     | varchar(50)  |
```
Comment (SimonLab) [inline README.md]: category-id and tag-id reference the keys id of the tags and categories so the type should be Integer (serial use this type I think but I will check again)

Comment (JMurphyWeb) [inline README.md]: Type INTEGER should be fine, I tested it this morning.

**Comment by JMurphyWeb** [comment]
@mk4111 @SimonLab Given that our first iteration will not include the categories or the relations tables, can i propose that we take the second half of @mk4111s README addition and pop it into the issue for further discussion?  That way the README will not be 'ahead' of our project too far and we can have a brief but necessary discussion on table names.

Let me know what you think.

**Comment by SimonLab** [comment]
Ok let's just add the tags (and wait for the categories on the next iteration)

**Comment by mk4111** [comment]
@SimonLab - made the updates! Could you please review and if happy, merge it in? Thank you!

**Pull request merged**